docs/source/install/ewc_ha.rst

```diff
@@ -0,0 +1,286 @@
+|bwc| HA Cluster - BETA
+=======================
+
+|bwc| (EWC) is the commercial version of the StackStorm automation platform. EWC adds priority
+support, advanced features such as fine-tuned access control, LDAP, and Workflow Designer. To
+learn more about |bwc|, get an evaluation license, or request a quote, visit `extremenetworks.com/product/workflow-composer
+<https://www.extremenetworks.com/product/workflow-composer/>`_.
+
+This document provides an installation blueprint for a Highly Availabile StackStorm Enterprise (|bwc|) cluster
+based on `Kubernetes <https://kubernetes.io/>`__, a container orchestration platform at planet scale.
```

Comment:
Sorry.. still doesn't read quite right. Perhaps "for planet scale"?

Reply:
https://open.nytimes.com/agrarian-scale-kubernetes-part-1-7ea703d4de08

> Visit the Kubernetes homepage, and one of the first things you’ll see is the promise of “Planet Scale.”

How about "of planet scale"?

Reply:
I would just not bother with the whole last part. Just link to Kubernetes, let them do their own description.